### ISSUE TITLE
Isomorphism Check is basically useless

### ISSUE THREAD

**Issue by fgrunewald** [marrink-lab/polyply_1.0#32]
https://github.com/fgrunewald/polyply_1.0/blob/e3d0148de2ea4ce4bdd92404937a6eba5197212d/polyply/src/apply_links.py#L304-L318

This code actually (almost) always evaluates to true, because we already checked that all resids are part of the residue graph and a path is at most of length 4, we almost always find a fitting subgraph. Instead we should check that all nodes and edges are part of the meta-molecule graph or that the path is part of the graph.

@pckroon Do you know a nx function that would do this.

In a nutshell it should do:

```
for node in graph2.nodes:
        if not node in graph1.nodes:
           return False
for edge in graph2.edges:
        if not edge in graph1.edges:
           return False
return True
```
